config/core/test-configs.yaml

```diff
@@ -2388,6 +2394,11 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: kontron-kswitch-d10-mmt-8g
```

Comment:
The device type doesn't appear to be online in the lab yet:
https://lavalab.kontron.com/scheduler/device_types

Reply:
@mwalle can say something about the schedule when the board is expected to be in the lab.

Reply:
It looks it's still not present. @mwalle @hthiery Do you have any updates about it?

Reply:
Sorry for the late answer. Due to hardware shortage there are no spare prototypes at the moment. New batch is scheduled for the end of the month. But I'll have to negotiate if there is a free one for the kernelci. It's up to you if you want to close this PR and I'll reopen a new one once the hardware is available or if you will keep it open.

Reply:
Thanks for the detailed update.
I think it's fine to leave the PR open for some time. Let's see if the hardware shortage issue resolves anytime soon.

Reply:
Ahh, I messed up the reply. I made a new comment this afternoon, that the device is now online. That must have triggered a notification, but apparently you didn't see the latest reply ;) Anyway, the device is up and running.